egs/mgb5/s5/local/chain/tuning/run_tdnn_1b.sh

```diff
@@ -0,0 +1,277 @@
+#!/bin/bash
```

Comment:
add your name and mention where the files was copied from,.
Please update the comments (WER and infos)

Reply:
removed it as it is not used.